install/kubernetes/helm/istio/test-values/values-e2e.yaml

```diff
@@ -25,9 +25,6 @@ gateways:
       requests:
         cpu: 10m
         memory: 40Mi
-      limits:
-        cpu: 100m
-        memory: 128Mi
```

Comment:
If our e2e tests are using this much resource I think we would want them to die, since something probably went wrong